src/components/Heading/Heading.tsx

```diff
@@ -20,6 +20,8 @@ export interface HeadingProps extends HTMLAttributes<HTMLHeadingElement> {
  * - `level` 속성을 통해서 `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>` 요소 중 하나를 선택할 수 있습니다.
  * - `level` 속성은 단계 별 기본 텍스트 스타일을 제공합니다.
  * - `size` 속성을 통해서 기본 스타일을 변경할 수 있습니다.
+ * - `level`과 `size` 속성이 같이 있는 경우, `size`속성이 `level`속성보다 우선순위가 높습니다.
+ * 예를 들어 `size`속성은 2, `level`속성이 1인 경우, `<h1>` 태그이지만 `size`속성이 2에 해당하는 스타일이 적용됩니다.
```

Comment:
```suggestion
 * 예를 들어 `size` 속성은 2, `level` 속성이 1인 경우, `<h1>` 태그이지만 `size` 속성이 2에 해당하는 스타일이 적용됩니다.
```

엄청 minor 한데요. 위에서는

> `size` 속성을 ... 

이어서 size와 속성 사이에 띄어쓰기가 있어요.
하지만 추가된 문장에서는 띄어쓰기가 없습니다. 그래서 문장이 일관적이지 않아 보였어요.

Reply:
@sounmind 크.. 예리하십니다. 네, 수정해서 올리겠습니다. 꼼꼼하게 봐주셔서 감사합니다! :)